SpatialGDK/Extras/internal-documentation/release-process.md

```diff
@@ -153,13 +158,11 @@ Copy the latest release notes from `CHANGELOG.md` and paste them into the releas
 1. In `UnrealGDKExampleProject`, merge `release` into `master`.
 
 **UnrealGDKTestGyms**
-1. In `UnrealGDKTestGyms`, merge `x.y.z-rc` into `preview`.
-1. If you want to soak test this release on the `preview`, use the [GitHub Release UI](https://github.com/spatialos/UnrealGDKTestGyms/releases) to tag the commit you just made to `preview` as `x.y.z-preview`.<br/>
-Copy the latest release notes from `CHANGELOG.md` and paste them into the release description field.
-1. In `UnrealGDKTestGyms`, merge `preview` into `release`.
-1. Use the [GitHub Release UI](https://github.com/spatialos/UnrealGDKTestGyms/releases) to tag the commit you just made to `release` as `x.y.z`.<br/>
-Copy the latest release notes from `CHANGELOG.md` and paste them into the release description field.
-1. In `UnrealGDKTestGyms`, merge `release` into `master`.
+1. Tag commit as `x.y.z`
```

Comment:
This should be tagged based on the RC or we merge rc into release and then tag release branch?
I assume the second since we made an RC in the first place.

Reply:
These repos don't and won't have release branches. This change is in reference to that.

Reply:
@io-ben